packages/server/src/routes/plans/create-master-plan copy.ts

```
@@ -0,0 +1,73 @@
+import { z } from "zod";
+import { createRoute } from "@hono/zod-openapi";
+import { AppRouteHandler } from "../../base/type";
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)